pkgs/development/libraries/protobuf/generic-v3-cmake.nix

```diff
@@ -46,7 +46,7 @@ let
         --replace 'tmpnam(b)' '"'$TMPDIR'/foo"'
     '';
 
-    patches = lib.optionals (lib.versionOlder version "3.22") [
+    patches = lib.optionals (lib.versionOlder version "22") [
```

Comment:
Does this still work like this?

Reply:
protobuf 3.21 was the only one using the cmake generic builder. So yes :).